File: All Codes/URI_1043.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
	
    float A,B,C;
    cin >> A >> B >> C;

    if (A+B>C && A+C>B && B+C>A)
        cout << "Perimetro = " << fixed << setprecision(1) << A+B+C << endl;
    else
        cout << "Area = " << fixed << setprecision(1) << ((A+B)*C)/2 << endl;

    return 0;
}

File: All Codes/URI_1043.py
a = [float(x) for x in input().split()]

if a[0] + a[1] > a[2] and a[0] + a[2] > a[1] and a[1] + a[2] > a[0]:
    print(f"Perimetro = {sum(a):.1f}")
else:
    print(f"Area = {((a[0] + a[1]) * a[2]) / 2:.1f}")